request/builder.go

```diff
@@ -38,6 +38,16 @@ func BuildRequest(unconfiguredRequest Request, profileNames []string, variables
 func configureRequest(unconfiguredRequest Request, profiles []profile.Options, passedVariables map[string]string) Request {
 	mergedProfile := profile.MergeOptions(profiles)
 
+	method := unconfiguredRequest.Method
+
+	if method == "" {
+		if unconfiguredRequest.Body == "" {
+			method = "GET"
```

Comment:
Can you use the [built-in HTTP constants](https://golang.org/pkg/net/http/#pkg-constants)?

Reply:
Definitely should. Didn't know about them.